autopush/web/webpush.py

```diff
@@ -315,27 +314,18 @@ def token_prep(self, d):
     def validate_auth(self, d):
         auth = d["headers"].get("authorization")
         needs_auth = d["token_info"]["api_ver"] == "v2"
-        if not auth and not needs_auth:
+        if not needs_auth and not auth:
             return
-
-        public_key = d["subscription"].get("public_key")
-        try:
-            auth_type, token = auth.split(' ', 1)
-        except ValueError:
-            raise InvalidRequest("Invalid Authorization Header",
-                                 status_code=401, errno=109,
-                                 headers={"www-authenticate": PREF_SCHEME})
-
-        # If its not a bearer token containing what may be JWT, stop
-        if auth_type.lower() not in AUTH_SCHEMES or '.' not in token:
-            if needs_auth:
-                raise InvalidRequest("Missing Authorization Header",
-                                     status_code=401, errno=109)
-            return
-
         try:
+            token = self.context['settings'].vapid_auth['t']
```

Comment:
I don't understand how this got here, it's not being carried around on the global object, is it?